### DISCUSSION TITLE
LDAP, can`t login under Administrator

### DISCUSSION THREAD

**Discussion by onixd** [requarks/wiki#4708] [Error / Bug Report]
I have LDAP auth in settings, when I try to login under Administrator:
Cannot read properties of undefined (reading 'key')

In login page where are no button "local"

WikiJS: 2.5.219 (docker)
DB: postgres

**Comment by mulfvik** [comment]
Perhaps this is the cause to your problem @onixd ?
When there is no authenticator picked (turns to a blueish color, the default behavior is that one authenticator is picked for you), the login module returns undefined if no strategy is defined and that error is rendered in the message box.
It would be nice to get a "Please select an strategy (or authenticator)" back if there is no strategy selected.
